project/Dependencies.scala

```diff
@@ -12,45 +12,36 @@ import sbt._
 object Dependencies {
 
   object Versions {
-    val crossScala   = Seq("2.13.3", "2.12.12")
+    val crossScala = Seq("2.13.3", "2.12.12")
```

Comment:
Minor: This file used to be formatted correctly, regarding spacing, but now it's not

Reply:
See the styling PR which you approved, i wanted to keep styling and logic changes seperate